vizro-core/tests/unit/vizro/conftest.py

```diff
@@ -44,5 +43,3 @@ def dashboard(page1, page2):
 @pytest.fixture()
 def dashboard_prebuild(dashboard):
     yield dashboard.pre_build()
```

Comment:
If we don't clean up after, let's change to a return instead of yield? Maybe double-check in other places too